crates/holochain/src/conductor/chc.rs

```diff
@@ -131,7 +131,7 @@ mod tests {
 
         let config = ConductorConfig {
             chc_url: Some(url2::Url2::parse(CHC_LOCAL_MAGIC_URL)),
-            ..Default::default()
+            ..ConductorConfig::empty()
```

Comment:
Again, I don't understand why not just default.

Reply:
removed.